### PR TITLE
This PR fixes the tests

### PR DESCRIPTION
**Pull request by gpolitis**
There have been a couple of API changes recently that broke the tests. This PR aims to fix that.

### CODE REVIEW

**Comment by gpolitis** [comment]
@andrei-gavrilescu could you please take a look?

File: src/features/extract.js [hunk 1]
```diff
@@ -52,9 +52,7 @@ if (isMainThread) {
     const clientid = process.argv[2];
 
     if (!clientid) {
```
Comment (andrei-gavrilescu) [inline src/features/extract.js:54]: just FYI, extract is no longer maintained at this point, we should move it to a deprecated folder, or something along those lines.

File: src/app.js [hunk 2]
```diff
@@ -85,11 +85,9 @@ async function persistDumpData(sinkMeta) {
     const { clientId } = sinkMeta;
     let uniqueClientId = clientId;
 
-    if (saveEntryAssureUnique) {
```
Comment (andrei-gavrilescu) [inline src/app.js:88]: from what I recall the flag was here for cases where dynamo is not configured (such as running in debug mode), so we don't send if we don't have an instance. A more elegant solution would have been nice, but for now I think we need some sort of guard there.

File: src/store/dynamo.js [hunk 3]
```diff
@@ -4,10 +4,6 @@ const dynamoose = require('dynamoose');
 const logger = require('../logging');
 const PromCollector = require('../metrics/PromCollector');
 
-if (!config.dynamo.tableName) {
```
Comment (andrei-gavrilescu) [inline src/store/dynamo.js:7]: Same comment as for saveEntryAssureUnique, we need some sort of check when dynamo is not configured.

Comment (andrei-gavrilescu) [reply src/store/dynamo.js:7]: nvm, just saw the added check to saveEntryAssureUnique.

File: src/app.js (hunk 2 above)
Comment (andrei-gavrilescu) [reply src/app.js:88]: nvm, just saw the added check to saveEntryAssureUnique.

File: .github/workflows/node.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16'
+    - run: npm install
+    - run: npm test
```
Comment (andrei-gavrilescu) [inline .github/workflows/node.yml:20]: Awesome!

**Review by andrei-gavrilescu — APPROVED** [review]

File: src/features/extract.js (hunk 1 above)
Comment (gpolitis) [reply src/features/extract.js:54]: Good point, I can do that in an upcoming PR

**Pull request merged**